packages/calling-component-bindings/src/participantListSelector.ts

```diff
@@ -25,18 +25,28 @@ const convertRemoteParticipantsToParticipantListParticipants = (
   const conversionCallback = (memoizeFn) => {
     return (
       remoteParticipants
-        // temporarily hiding lobby participants in ACS clients till we can admit users through ACS clients
+        /**
+         * hiding participants who are inLobby, idle, or connecting in ACS clients till we can admit users through ACS clients.
+         * phone users will be in the connecting state until they are connected to the call.
+         */
         .filter((participant: RemoteParticipantState) => {
-          return participant.state !== 'InLobby';
+          return (
+            (participant.state !== 'InLobby' && participant.state !== 'Idle' && participant.state !== 'Connecting') ||
+            participant.identifier.kind === 'phoneNumber'
+          );
         })
         .map((participant: RemoteParticipantState) => {
           const isScreenSharing = Object.values(participant.videoStreams).some(
             (videoStream) => videoStream.mediaStreamType === 'ScreenSharing' && videoStream.isAvailable
           );
+          const state =
+            participant.identifier.kind === 'phoneNumber' && participant.state === 'Connecting'
```

Comment:
do we need to handle Idle state here as well?

Reply:
no like with teams and group calls `idle` is acting like a placeholder state here. so we want to still show it like before even for PSTN participants.